### PR TITLE
test

### PR DESCRIPTION
**Pull request by robert082492**
testing

### CODE REVIEW

File: Makefile
Comment (anhphan21) [inline Makefile:1]: for LDFLAGS, you need to keep -static -O2 -Wall -g for debugging with gdb
You dont have to include the header file, only .cpp files are needed

Comment (anhphan21) [reply Makefile:1]: Actually, the sources can be re-written like this SOURCES= src*.cpp

**Review by anhphan21 — COMMENTED** [review]
I left some comment on Makefile

File: Makefile
Comment (robert082492) [reply Makefile:1]: why should it be src*.cpp?

Comment (anhphan21) [reply Makefile:1]: I miss slash, it should be like this src/*.cpp

**Pull request merged**